ftest.sh

```diff
@@ -45,6 +45,7 @@ if [ -f .localenv ]; then
 fi
 
 TEST_TAG="${1:-quick}"
+TEST_DIR="/tmp/Functional_${TEST_TAG//[, ]/-}"
```

Comment:
FWIW, spaces and commas are legal values in pathnames.  By replacing both with a single value, you are obscuring which tags were _and_'d and which were _or_'d.

Leaving the spaces and commas alone should be fine if your variable expansions below follow [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086) recommendations.

Reply:
Thanks for the great feedback.  Fix applied.

Reply:
I ended up reverting back to replacing just the space with a dash as the python test scripts that use this path do not account for spaces in the path.